Proposals/ParametricandOpticalAxes/ProposalSummary.md

```diff
@@ -0,0 +1,373 @@
+
+# Proposal: Parametric and Optical Axes
+
+## Administrative Information
+**Proposers name:** Sam Berlow
+
+**Vendor affiliation:** Type Network
+
+**Proposal name:** Parametric Axes
+
+**Date of submission:** 12/6/2017
+
+**New or revised proposal:** New
+
+**Previous revision date:** N/A
+
+## General Technical Information
+
+**Related axes:** See individual axis proposals
+
+**Similar axes:** See individual axis proposals
+
+**Axis type:** Parametric
+
+## Proposed Axis: `xtra`
+
+**Tag:** xtra
+
+**Name:** x transparent
+
+**Axis type:** Optical
+
+**Description:** assigns a “white” per mille value to each instance of the design space
+
+**Valid numeric range:**  -1000 to 2000
+
+**Scale interpretation:** Values should be interpreted as per-mille-of-em
+
+**Recommended or required “Regular” value:** N/A
+
+**Suggested programmatic interactions:** Example: by setting a maximum word space parameter, and adjusting the XTRA units to a min and max. A developer could adjust justified text without tracking letters. Maintaining the apparent weight and width. 
+
+**UI recommendations:** Users may choose to program a variant in connection to direct or conjunctive input for a page description language, or via a user interface
+
+**Script or language considerations:** Many languages and scripts 
```

Comment:
Are there any special considerations that might be needed for certain scripts? Or will it be relevant for some scripts but not others? If no special considerations and relevance isn't limited to certain scripts, then just say "Can be used for all scripts".